### PR TITLE
Fields Cleanup

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: metricbeat/docs/modules/system/network.asciidoc [hunk 1]
```diff
@@ -38,30 +38,38 @@ Below is an example document generated by this metricset.
...
-            "bytes_recv": 15105407377,
-            "bytes_sent": 4101433502,
-            "dropin": 0,
-            "dropout": 17971,
-            "errin": 0,
-            "errout": 0,
-            "name": "en0",
-            "packets_recv": 19064644,
-            "packets_sent": 14230416
+            "in": {
```
Comment (monicasarbu) [inline metricbeat/docs/modules/system/network.asciidoc]: nice!

**Comment by monicasarbu** [comment]
We still need to do the following changes:

- system.memory.memory.*.used_pct -> system.memory.memory.*.used.pct
- it's strange to have system.memory.memory and system.memory.swap (see below) -> we need a better name
```
  },
  "system": {
    "memory": {
      "memory": {
        "actual": {
          "free": 5169119232,
          "used": 12010749952,
          "used_pct": 0.6991
        },
        "free": 4154376192,
        "total": 17179869184,
        "used": 13025492992,
        "used_pct": 0.7582
      },
      "swap": {
        "free": 1709703168,
        "total": 2147483648,
        "used": 437780480,
        "used_pct": 0.2039
      }
    }
```
- change `total_p` to `total.pct` in `system.process`:
```
 "system": {
    "process": {
      "cmdline": "/sbin/launchd",
      "cpu": {
        "start_time": "May26",
        "total_p": 0
      },
      "mem": {
        "rss": 9859072,
        "rss_p": 0.0006,
        "share": 0,
        "size": 2554535936
      },
      "name": "launchd",
      "pid": 1,
      "ppid": 0,
      "state": "running",
      "username": "root"
    }
```

**Comment by monicasarbu** [comment]
Another one:
- used_p -> used.pct:

```
 "system": {
    "filesystem": {
      "avail": 0,
      "device_name": "map auto_home",
      "files": 0,
      "free": 0,
      "free_files": 0,
      "mount_point": "/home",
      "total": 0,
      "used": 0,
      "used_p": 0
    }
```

**Comment by ruflin** [comment]
For the memory: What if we move it to the top level:
```
"system": {
    "memory": {
      "actual": {
        "free": 5169119232,
        "used": 12010749952,
        "used_pct": 0.6991
      },
      "free": 4154376192,
      "total": 17179869184,
      "used": 13025492992,
      "used_pct": 0.7582
      "swap": {
        "free": 1709703168,
        "total": 2147483648,
        "used": 437780480,
        "used_pct": 0.2039
      }
    }
```

For the missing user_pct. The problem is that these conflict with user. So the option would be user.value and user.pct.

**Comment by ruflin** [comment]
@monicasarbu I restructured the memory structure to the above. For the `used` and `used_pct` fields I suggest we leave it for the moment. I fixed the once that were still _p

File: metricbeat/docs/modules/system/network.asciidoc (hunk 1 above)
Comment (andrewkroh) [inline metricbeat/docs/modules/system/network.asciidoc]: I think the received metrics should be put under `in` and the sent under `out`.

Comment (ruflin) [inline metricbeat/docs/modules/system/network.asciidoc]: Changed

**Pull request merged**